available_courses.md

```diff
@@ -93,3 +93,22 @@ Or
 - You might be familiar with version control and basic web architecture
 
 Donor: [Todd McLeod](https://bit.ly/Todd_McLeod_LHTCG)
+
+
+
+### Learn How To Code Google's Go (golang) Programming Language
+More details: [Go for Complete Beginners](https://github.com/Pisush/go_for_complete_beginners)
+
```

Comment:
https://github.com/Pisush/go_for_complete_beginners/blob/master/course_outline.md

Are you planning to move this to the readme?